frontend/src/app/testQueue/QueueItem.tsx

```diff
@@ -120,39 +120,41 @@ const UPDATE_AOE = gql`
 `;
 
 interface AreYouSureProps {
-  patientName: string;
+  cancelText: string;
+  continueText: string;
   cancelHandler: () => void;
   continueHandler: () => void;
 }
 
 const AreYouSure: React.FC<AreYouSureProps> = ({
-  patientName,
   cancelHandler,
+  cancelText,
   continueHandler,
+  continueText,
+  children,
```

Comment:
This is pretty close to being a truly generic okay-cancel wrapper around the modal, I wonder if it makes sense to take it all the way and make it a common component. Something like e.g.
```
<OkayCancelDialog 
     okayLabel="OK, delete" 
     cancelLabel="No, do nothing" 
     onOkay={()=> blowItAway()} 
     onCancel={() => setShowDialog(false)}>
  <h2>BIG HEADING</h2>
  <p>Moar content</p>
</OkayCancelDialog>
```

Reply:
I do think we need to do this eventually, but I kind of want to punt it to an overall Modal refactor